middleware/gateway/include/gateway/group/inbound/state.h

```diff
@@ -339,8 +212,8 @@ namespace casual
             CASUAL_SERIALIZE( directive);
             CASUAL_SERIALIZE( external);
             CASUAL_SERIALIZE( pending);
-            CASUAL_SERIALIZE( correlations);
-            CASUAL_SERIALIZE( conversations);
+            //CASUAL_SERIALIZE( correlations);
+            //CASUAL_SERIALIZE( conversations);
```

Comment:
Remove

Reply:
There is another case of a commented CASUAL_SERIALIZE a bit up in the file (around line 174)